src/bundle/Resources/config/value_object_visitors.yml

```diff
@@ -1,47 +1,7 @@
-parameters:
-
-    # Errors
-
-    # Section
-
-    # URLWildcard
-
-    # URLAlias
-
-    # Content
-
-    # User group
-
-    # User
-
-    # ContentType
-
-    # Object state
-
-    # View
-
-    # Trash
-
-    # Location
-
-    # Roles
-
-    # Relation
-
-    # Field Definition
-
-    # Content type Group
-
-    # HTTP
-
-    # Cache
-
-    # Services
-
```

Comment:
Isn't that more of "Table of Contents", so you can see what is where? I have no idea why someone did it this way instead of splitting it into separate configurations but that was the purpose afaik.

Reply:
But this have totally no value under empty `parameters` section, even more when you see that `Services` are last.
Services have their own titles when given section starts. from my pov this is totally useless and just clutters config file.

Reply:
It is not useless, but it is stupid beyond your imagination... you copy header name and look for other occurrences in file to jump to section you need. :D

Reply:
But this is so inconsistent - there are `#Roles` and `#Role`, `#Content type Group` is not even there, it more like looks that there were some parameters before - maybe those `.class` ones which where used before rebranding and thats whats left after we dropped them.